### PR TITLE
Fixes #14364: Add support to multiple centeon pollers

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14364

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: centreon/bin/centreon-plugin [hunk 1]
```diff
@@ -84,7 +85,11 @@ def dictifyNode(node):
         if validateIPAddr(ip):
             ipAddr = ip
             break
-    return { 'hostname': node['hostname'], 'rudder_id': node['id'], 'node_type': 'server' if node['id'] == 'root' else 'agent', 'ip_address': ipAddr }
+    return { 'hostname': node['hostname'], 
+             'rudder_id': node['id'],
+             'node_type': 'server' if node['id'] == 'root' else 'agent', 
+             'ip_address': ipAddr,
+             'relay': node['policySErverId'] }
```
Comment (amousset) [inline centreon/bin/centreon-plugin]: policyServerId

File: centreon/bin/centreon-plugin [hunk 2]
```diff
@@ -257,32 +288,30 @@ def applyRudderMonitoringConfigurations(conf):
...
             #register[dir]['templates'] = ','.join(list(set(register[dir]['templates'].split(','))))
 
             # update hash value
             hash_list[dir] = hash_value
 
-    if changed:
-        register.write(open(registerFile, 'w'))
-        print('[ ] Reloading Centreon poller config, restarting poller...')
-        Webservice.getInstance().restart_poller(conf.get('CENTREON', 'centreonPoller'))
+    poller_list = restart_pollers(all_pollers, poller_list)
```
Comment (amousset) [inline centreon/bin/centreon-plugin]: all_pollers does not seem defined

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**